tests/collab/stress_test.rs

```diff
@@ -0,0 +1,64 @@
+use std::sync::Arc;
+use std::time::Duration;
+
+use collab_entity::CollabType;
+use serde_json::json;
+use tokio::time::sleep;
+use uuid::Uuid;
+
+use client_api_test::{assert_server_collab, TestClient};
+
+use super::util::TestScenario;
+
+#[tokio::test(flavor = "multi_thread", worker_threads = 4)]
+async fn run_multiple_text_edits() {
+  let test_scenario = Arc::new(TestScenario::open(
+    "./tests/collab/asset/automerge-paper.json.gz",
+  ));
+  let mut tasks = Vec::new();
+  for _i in 0..1 {
```

Comment:
do we need to increase the number?

Reply:
It's still not passing, I'm running into issues with ACKs. What's worse none of the present tests found these issues.